Error when running "npm run dev" related to missing parameterData.json file

#### Most appropriate sub-area of p5.js?

- [X] Build tools and processes

#### Details about the bug:

- p5.js version: commit 22c858186f0b8e3ae12152171012aa341e976d95
- Operating System: macOS 10.15.7

#### Steps to reproduce this:

- follow instructions at https://p5js.org/contributor-docs/
- check out version 22c858186f0b8e3ae12152171012aa341e976d95
- run `npm ci`
- run `npm run dev`

Error shown is:

```
$ npm run dev

> p5@1.4.0 dev
> grunt browserify:dev connect:yui watch:quick

Running "browserify:dev" (browserify) task
Fatal error: Cannot find module '../../../docs/parameterData.json' from '<...local path...>/p5.js/src/core/friendly_errors'
```

#### Solution:

Running `npm run grunt` once before running `npm run dev` fixes the issue as it generates the missing `parameterData.json` file.

I believe that the documentation should be updated to reflect this i.e. "make sure to run `npm run grunt` at least once before running `npm run dev`"

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "scripts": {
     "grunt": "grunt",
     "build": "grunt build",
-    "dev": "grunt browserify:dev connect:yui watch:quick",
+    "dev": "grunt yui browserify:dev connect:yui watch:quick",
     "docs": "grunt yui",
     "docs:dev": "grunt yui:dev",
     "test": "grunt",

